index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Rust Argentina</title>
+
+    <base data-trunk-public-url/>
+    <link data-trunk rel="css" href="src/styles/tailwind.css"/>
+    <link data-trunk rel="css" href="src/styles/components.css"/>
```

Comment:
Any assets/statics should be added here and they will be mapped with hash names into the `/dist` directory when building